package.json

```diff
@@ -5,7 +5,7 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "eject": "expo eject",
-    "test": "./node_modules/.bin/jest --watch"
+    "test": "./node_modules/.bin/jest"
```

Comment:
Watch is helpful b/c it allows you to save changes and auto run the tests. Why did you remove it?